presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergPageSink.java

```diff
@@ -119,7 +143,13 @@ public IcebergPageSink(
             JsonCodec<CommitTaskData> jsonCodec,
             ConnectorSession session,
             FileFormat fileFormat,
-            int maxOpenWriters)
+            int maxOpenWriters,
+            List<SortField> sortOrder,
+            DataSize sortingFileWriterBufferSize,
+            int sortingFileWriterMaxOpenFiles,
+            TypeManager typeManager,
+            PageSorter pageSorter,
+            OrcFileWriterFactory orcFileWriterFactory)
```

Comment:
There are quite a few fields related to sorting. I'm wondering if it might make sense to put them in their own class together.

Also, digging deeper, I realized that this seems to only be supported on ORC files due to the sorting file writer API. Have we considered enhancing this to support parquet?

Reply:
@ZacBlanco It also supports sorting data in Parquet files.

Do you really want me to do that class changes  in this PR or can i open separate git issue to track this changes?